x-pack/test/functional/services/ml/api.ts

```diff
@@ -1205,13 +1205,57 @@ export function MachineLearningAPIProvider({ getService }: FtrProviderContext) {
       log.debug('> Trained model definition uploaded');
     },
 
+    async getTrainedModelsES() {
+      log.debug(`Getting trained models`);
+      const { body, status } = await esSupertest.get(`/_ml/trained_models`);
+      this.assertResponseStatusCode(200, status, body);
+
+      log.debug('> Trained models fetched');
+      return body;
+    },
+
     async deleteTrainedModelES(modelId: string) {
-      log.debug(`Creating trained model with id "${modelId}"`);
-      const { body: model, status } = await esSupertest.delete(`/_ml/trained_models/${modelId}`);
-      this.assertResponseStatusCode(200, status, model);
+      log.debug(`Deleting trained model with id "${modelId}"`);
+      const { body, status } = await esSupertest
+        .delete(`/_ml/trained_models/${modelId}`)
+        .query({ force: true });
+      this.assertResponseStatusCode(200, status, body);
 
-      log.debug('> Trained model created');
-      return model;
+      log.debug('> Trained model deleted');
+    },
+
+    async deleteAllTrainedModelsES() {
+      log.debug(`Deleting all trained models`);
+      const getModelsRsp = await this.getTrainedModelsES();
+      for (const model of getModelsRsp.trained_model_configs) {
+        if (model.model_id === 'lang_ident_model_1') {
```

Comment:
maybe we should have a function for this check?
Just in case this list expands.
```
function isInternalModal(model: TrainedModelConfigResponse) {
  return ['lang_ident_model_1'].includes(model.model_id);
}
```

Reply:
Good point! Added in 2a818b3